### ISSUE TITLE
NTR: Elevation in beta human chorionic gonadotropin in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8417]
New term request Elevation in beta human chorionic gonadotropin in blood (nadea3_181121023608): beta HCG  UMLS: 1735879

**Comment by pnrobinson** [comment]
@MickeySegal
just to ask whether this is distinct from
[High maternal circulating chorionic gonadotropin concentration HP:0011433](https://hpo.jax.org/app/browse/term/HP:0011433)?

**Comment by MickeySegal** [comment]
Not just maternal, also from tumors:  https://pubmed.ncbi.nlm.nih.gov/11919246/ "Nonseminomatous histology, presence of nonpulmonary visceral metastases, primary mediastinal GCT location, and elevated beta-human chorionic gonadotropin were independent prognostic factors for shorter survival"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000485
- label: Elevated circulating beta chorionic gonadotropin concentration
- parents: [Abnormal circulating protein concentration (HP:0010876)](https://hpo.jax.org/app/browse/term/HP:0010876)
- pmid: PMID:11919246
- def: The concentration of beta chorionic gonadotropin in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8417
- synonyms:
